### ISSUE TITLE
Vizualization

### ISSUE THREAD

**Issue by nathanielsimard** [nathanielsimard/improving-fs-ssl#33]
## For each task
* training loss
* validation loss
* training metric
* validation metric

## For testing

* aggregated accuracy across different support/query set with variance!
